### PR TITLE
Fix for File Pointer Resource Leak in Application write_remove_test

### PR DESCRIPTION
**Pull request by samirsahu1947**
Fix for File Pointer Resource Leak in Application write_remove_test

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

File: apps/examples/fs_helper/write_remove_test.c [hunk 1]
```diff
@@ -53,17 +53,16 @@ static int make_file(char *filename)
...
+			fclose(fp);
+			return -1;
 		}
 	}
 
-	ret = fclose(fp);
-	if (ret != 0) {
-		printf("fclose error[%d]\n", ret);
-		return ret;
+	if (fclose(fp) != 0) {
```
Comment (Taejun-Kwon) [inline apps/examples/fs_helper/write_remove_test.c]: you should keep ret because it checks ret as a return value.

**Comment by seinfra** [comment]
Code Rule Check OK.

File: apps/examples/fs_helper/write_remove_test.c [hunk 2]
```diff
@@ -53,17 +53,16 @@ static int make_file(char *filename)
 	}
 
 	for (i = 0; i < WRITE_DATA_SIZE; i++) {
-		ret = fwrite(&i, 4, 1, fp);
-		if (ret < 0) {
+		if (fwrite(&i, 4, 1, fp) < 0) {
 			printf("fwrite error[%d]\n", ret);
```
Comment (pradeepns85) [inline apps/examples/fs_helper/write_remove_test.c]: printf will print previous / old value as fwrite error because it's not been set now .

**Pull request merged**